src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  }
+
+  return days * 40;
 }
```

Comment:
The function `calculateRentalCost` is correctly implemented and is following the task requirements. However, there are some improvements that can be made to increase code readability and maintainability.